_reports/adhikasp.md

```diff
@@ -40,104 +40,129 @@ blog: https://adhikasp.github.io
 activity:
  - 0:
    - repo: coala
-   - link: https://github.com/coala/coala/pull/4412
+   - link: https://github.com/coala/coala/pull/4412/commits/66d0276f0a45f221bc2d880a4fd5d4c63f6d119e
    - details: >
       aspectModule: Add ``get()`` method
  - 1:
    - repo: coala
-   - link: https://github.com/coala/coala/pull/4392
+   - links: https://github.com/coala/coala/pull/4392/commits/0a67a087f1832c7500b8e5ba362c50546e0bd9fa
+            https://github.com/coala/coala/pull/4392/commits/79b18a953d32ecc14b00441fb272717f7d4b7f72
+            https://github.com/coala/coala/pull/4392/commits/705d256e8344d676e89c0f82c8a4e0a86934e197
```

Comment:
Sorry, my review wasn't very clear when I said 'you should have 3 items here'. 
This is wrong; See https://deploy-preview-656--coala-projects.netlify.com/gsoc/2017/statusreport/adhikasp. Our template doesn't support `links` atm. Instead, you should create three `activity` here instead of three links within one activity.
e.g.:
```yaml
 - 1:
   - repo: coala
   - link: https://github.com/coala/coala/pull/4392/commits/0a67a087f1832c7500b8e5ba362c50546e0bd9fa
   - details: >
      aspect test: Add SubSubAspect fixture
 - 2:
   - repo: coala
   - link: https://github.com/coala/coala/pull/4392/commits/79b18a953d32ecc14b00441fb272717f7d4b7f72
   - details: >
      aspectbase: Add ``get()`` method
```

Reply:
ok, i'm working on it